docs/ref/settings.txt

```diff
@@ -457,15 +457,24 @@ should be ``'HTTP_X_XSRF_TOKEN'``.
 
 Default: ``[]`` (Empty list)
 
-A list of hosts which are trusted origins for unsafe requests (e.g. ``POST``).
+A list of trusted origins for unsafe requests (e.g. ``POST``).
+
 For a :meth:`secure <django.http.HttpRequest.is_secure>` unsafe
 request, Django's CSRF protection requires that the request have a ``Referer``
 header that matches the origin present in the ``Host`` header. This prevents,
```

Comment:
(Diff suggestion doesn't really work here...) 

Should this say

```
...requires that the request have an ``Origin`` or ``Referer`` header that matches...
```

?

Reply:
Not exactly, but you're right this section needs reworking which I have done.